cypress/integration/layerErrorIcon.spec.js

```diff
@@ -0,0 +1,59 @@
+describe('Broken icon for layer errors', function() {
+  it('shows broken icon on a layer error when layer is added to map', function() {
+    cy.openWindow('/example/index.html#lat=41.624&lon=-91.626&zoom=5&layers=Standard')
+    cy.wait(300)
+    cy.window().then((win) => {
+      cy.stub(win.FracTracker_mobile, 'requestData')
+      cy.get('#menu-fracTrackerMobile label').click({ force: true }).then(() => {
+        win.FracTracker_mobile.onError('fracTrackerMobile')
```

Comment:
I checked the feature by stubbing the layer and manually calling the method instead.